azure-pipelines.yml

```diff
@@ -26,6 +33,21 @@ jobs:
   steps:
   - template: ci/publish-steps.yml
 
+# Courtesy push on Windows
+- job: courtesy_push_windows
```

Comment:
Nit: this can just be `courtesy_push` as we only have one.

Reply:
Will update

Reply:
Oh wait, I actually did this to maintain consistency w/ the publish step. I'd probably prefer to leave it, it doesn't show up in the display name anyways